client/App.jsx

```diff
@@ -80,6 +81,23 @@ class App extends Component {
     this.setState({ drawerShowing: null });
   }
 
+  beginQrScan(imageBufferData) {
```

Comment:
Before this gets merged in, I'll move this out into a vanilla JS module so it's not entangled with the react-ness at all.